package/MDAnalysis/analysis/gnm.py

```diff
@@ -402,18 +395,13 @@ def __init__(self,
                  cutoff=4.5,
```

Comment:
Can you also add a versionchanged here too please?

Reply:
Of course.